react-hyf/react-hyf-client/src/components/filter/GenderContainer.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { IconRandom, IconMale, IconFemale } from '../Icons';
+
+const GenderContainer = props => {
+  const isActive = gender => (gender === props.gender ? ' active' : '');
+  const [randomButton, maleButton, femaleButton] = props.buttons;
+  return (
+    <div className="gender">
+      <button
+        ref={randomButton}
```

Comment:
Why are you using `ref` here? Am I right that you are using it to 'fake click' the button when a key on the keyboard is pressed?

Reply:
I create ref buttons variables inside the parent component. Then I passed the variables to the children via props. Because I need to connect each ref buttons to the concerned button elements. That's why I use `ref` here.